frontend/src/api/network/servingRuntimes.ts

```diff
@@ -83,6 +58,22 @@ const assembleServingRuntime = (
   return updatedServingRuntime;
 };
 
+export const fetchServingRuntimeConfig = (configNamespace: string): Promise<ServingRuntimeKind> => {
+  return getConfigMap(configNamespace, 'servingruntimes-config').then((configmap) => {
+    const overrideServingRuntime = YAML.parse(configmap.data?.['override-config'] || '');
+    if (overrideServingRuntime) {
+      return overrideServingRuntime;
+    }
+    const defaultServingRuntime = YAML.parse(configmap.data?.['default-config'] || '');
+    if (defaultServingRuntime === null) {
+      throw new Error(
+        'ConfigMap `servingruntimes-config` is misconfigured or key might be missing from the body, fallback configuration will be applied',
```

Comment:
You throw this error -- the catch statement is just setting the error -- will the fallback be used?

Reply:
Yeah, since the result will have `null` value, and it will get the fallback configuration.